src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -1031,18 +1019,12 @@ public void onPlayerDamageByPlayer(final EntityDamageByEntityEvent e) {
 
 	@EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
 	public void onPlayerDeath(final PlayerDeathEvent event) {
-		if (!(event.getEntity() instanceof Player)) {
-			return;
-		}
-
 		if (event.getKeepInventory()) {
 			if (TempArmor.hasTempArmor(event.getEntity())) {
 				for (final TempArmor armor : TempArmor.getTempArmorList(event.getEntity())) {
 					armor.revert(event.getDrops());
 				}
 			}
-		} else {
-			// Do nothing. TempArmor drops are handled by the EntityDeath event and not PlayerDeath.

```

Comment:
This is perhaps a useful comment, it didn't need to be in an empty else though idk.